frontend/src/components/pages/rp-connect/secrets/Secrets.Update.tsx

```diff
@@ -0,0 +1,134 @@
+import {Button, ButtonGroup, createStandaloneToast, Flex, FormField, Input} from '@redpanda-data/ui';
+import {PageComponent, PageInitHelper} from '../../Page';
+import {observer} from 'mobx-react';
+import {appGlobal} from '../../../../state/appGlobal';
+import {pipelinesApi, rpcnSecretManagerApi} from '../../../../state/backendApi';
+import PageContent from '../../../misc/PageContent';
+import {action, makeObservable, observable} from 'mobx';
+import {DefaultSkeleton} from '../../../../utils/tsxUtils';
+import {formatPipelineError} from '../errors';
+import {Scope, UpdateSecretRequest} from '../../../../protogen/redpanda/api/dataplane/v1alpha2/secret_pb';
+
+const {ToastContainer, toast} = createStandaloneToast();
+
+const returnSecretTab = '/connect-clusters?defaultTab=redpanda-connect-secret'
+
+@observer
+class RpConnectSecretUpdate extends PageComponent<{ secretId: string }> {
+    @observable secret = '';
+    @observable isUpdating = false;
+
+    constructor(p: any) {
+        super(p);
+        makeObservable(this, undefined, {autoBind: true});
+    }
+
+    initPage(p: PageInitHelper) {
+        p.title = 'Update Secret';
+        p.addBreadcrumb('Redpanda Connect Secret Manager', '/rp-connect/secret/update');
```

Comment:
Note: I think in the future we should have a global place with all the routes to ensure we can easily change name/forbid accessing it across the app if we needed to. Just in the router file there should be a place to get all the route paths.

Reply:
I don't know what was the reason for this change, we can discuss it later

Reply:
Should it be secret or secrets?

Reply:
it should be secrets, I fixing it